src/python/grpcio/grpc/_simple_stubs.py

```diff
@@ -268,15 +301,23 @@ def unary_unary(
         unset, defaults to 60 seconds. Supply a value of None to indicate that
         no timeout should be enforced.
       metadata: Optional metadata to send to the server.
+      _registered_method: INTERNAL USE ONLY. Whether this RPC is made for a
```

Comment:
Let's not document this in the public-facing docstrings. If we want to keep the comment, let's add it a few lines below.